ivre/data/scanners.py

```diff
@@ -70,3 +70,23 @@
+            "Nmap"
```

Comment:
I think you could add another element to the tuple, such as "top 1000 ports" here...
```suggestion
            "Nmap",
            "Top 1000 ports (default)",
```